test-bitcoincore-rpc/src/lib.rs

```diff
@@ -285,6 +373,99 @@ impl Api for Server {
     }
   }
 
+  fn create_raw_transaction(
+    &self,
+    utxos: Vec<CreateRawTransactionInput>,
+    outs: HashMap<String, f64>,
+    locktime: Option<i64>,
+    replaceable: Option<bool>,
+  ) -> Result<String, jsonrpc_core::Error> {
+    assert_eq!(locktime, None, "locktime param not supported");
+    assert_eq!(replaceable, None, "replaceable param not supported");
+
+    let tx = Transaction {
+      version: 0,
+      lock_time: PackedLockTime(0),
+      input: utxos
+        .iter()
+        .map(|input| TxIn {
+          previous_output: OutPoint::new(input.txid, input.vout),
+          script_sig: Script::new(),
+          sequence: Sequence(0),
+          witness: Witness::new(),
+        })
+        .collect(),
+      output: outs
+        .iter()
+        .map(|(_address, amount)| TxOut {
+          value: *amount as u64,
```

Comment:
Is this right? If amount is an `f64` amount of BTC, and `value` is an amount in sats, this should be `(amount * COIN_VALUE) as u64`.